.github/workflows/python-cross-platform-support.yml

```
@@ -0,0 +1,99 @@
+on:
+  workflow_call:
+
+defaults:
+  run:
+    working-directory: ./python
+
+jobs:
+  os_build_matrix:
+    name: Build ${{ matrix.os }} Secure Install Air Gap
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, ubuntu-latest, windows-latest, macos-latest] # Annoyingly, Github actions don't support anchors
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python 3.9.13
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9.13
+
+      - name: Install the AaC Specifications Package
+        run: python -m pip install -e
```

```suggestion
        run: python -m pip install -e .
```
Still need the `.`, that says where to find what to install. The `[all]` was what portion of the `setup.py`  to be used with the install from that location. Was a tag within the `setup.py` script.

---

Whoops, fixed.